### ISSUE TITLE
Properties file shouldn't contain real passwords or tokens

### ISSUE THREAD

**Issue by missaugustina** [kanesee/ai-github#1]
The properties file you check into the repo should be an example, like AppConfig.properties.sample and contain comments with instructions as well as default values.

It should not contain your actual Github token:
https://github.com/kanesee/ai-github/blob/master/AppConfig.properties#L1

Stuff like this really shouldn't go into this kind of file but should probably be set as environment variables or in a config file in the user's $HOME dir.

**Comment by missaugustina** [comment]
I have a fix for this but I had to rebase down to the root commit because you included your private info when you created the repo - https://github.com/missaugustina/ai-github/tree/issue1_ignore_properties

Github won't let me submit a PR to fix it.

$ git rebase -i --root HEAD
$ git rm AppConfig.properties
$ git add .
$ git rebase --continue
$ git rm AppConfig.properties
$ git rebase --continue
$ git checkout -b issue1_ignore_properties
$ vim .gitignore
$ vim AppConfig.properties
$ git commit -a

**Comment by kanesee** [comment]
Thank you for pointing that out. I've removed the file and its history (I
believe)

On Mon, Jun 26, 2017 at 9:09 PM, Augustina Ragwitz <notifications@github.com

**Comment by missaugustina** [comment]
Yeah it looks like you got it! I would also add the Appconfig.properties file to the .gitignore. It looks like you all might be committing to git through your Java IDE? I know Java projects usually have a lot of extra files scattered all over the place, so that could end up back in the repo on accident. I submitted a PR with the change.
